go/vt/vtgate/executor.go

```diff
@@ -493,13 +507,18 @@ func (e *Executor) handleBegin(ctx context.Context, safeSession *SafeSession, sq
 	logStats.PlanTime = execStart.Sub(logStats.StartTime)
 	err := e.txConn.Begin(ctx, safeSession)
 	logStats.ExecuteTime = time.Since(execStart)
+
+	queriesProcessed.Add("Begin", 1)
+
 	return &sqltypes.Result{}, err
 }
 
 func (e *Executor) handleCommit(ctx context.Context, safeSession *SafeSession, sql string, bindVars map[string]*querypb.BindVariable, logStats *LogStats) (*sqltypes.Result, error) {
 	execStart := time.Now()
 	logStats.PlanTime = execStart.Sub(logStats.StartTime)
 	logStats.ShardQueries = uint32(len(safeSession.ShardSessions))
+	queriesProcessed.Add("Commit", 1)
+	queriesRouted.Add("Commit", int64(logStats.ShardQueries))
```

Comment:
I thought about that as well and came to the conclusion it wasn't worth the cardinality increase, though I can certainly see where it would be useful.

I could get convinced otherwise.

Reply:
Oh good point about the cardinality increase. 🤔 

We can add them later if we want them! I say let's ship this and see what we get out of it.